### PR TITLE
Moved hydrater list to resources

### PR DESCRIPTION
**Pull request by Neamar**

### CODE REVIEW

**Comment by merlinND** [comment]
Doesn't this need to be compiled somehow? (as the endpoints' doc)
I don't know how are the pages being served.

File: resources/hydraters.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+Anyfetch ships with some default hydraters you may want to check.
+
+## File hydraters
+Hydraters working on binary files.
+
+### plaintext.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/plaintext.hydrater.anyfetch.com)
+
+Retrieve basic information about a file (word, pdf, ppt, txt, rtf, png...). Will extract text if available.
```
Comment (merlinND) [inline resources/hydraters.md:14]: > Will extract any text exposed by a file. It supports a variety of formats (word, pdf, ppt, txt, rtf, png...).

(If that's the case? Does it handle a lot of formats, or any format?)

**Comment by merlinND** [comment]
Could we credit the various tools we use in each hydrater?

File: resources/hydraters.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+* Output document type: `document`
+* Output metadata:
+    - `text`
+* Output data:
+    - `html` 
+
+### ocr.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/ocr.hydrater.anyfetch.com)
+
+Retrieve textual information from an image.
```
Comment (merlinND) [inline resources/hydraters.md]: Retrieve textual information from an image by running an OCR algorithm. Best results are obtained with scanned documents.

File: resources/hydraters.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+* Input document type: `file`
```
Comment (merlinND) [inline resources/hydraters.md:28]: Could we be more precise? Do images files have their own document type?

File: resources/hydraters.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+Transform a PDF file to HTML.
+
+* Input document type: `document`
+* Output data:
+    - `html`
+
+### office.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/office.hydrater.anyfetch.com)
+
+Transform an office file to PDF, inject it back onto anyFetch.
```
Comment (merlinND) [inline resources/hydraters.md]: inject it back into the **A** nyFetch hydration chain / or as a new document?

File: resources/hydraters.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+* Input document type: `document`
+* Output data:
+    - `html`
+
+### eml.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/eml.hydrater.anyfetch.com)
+
+Retrieve data from eml files, hydrate mail data and save back the attachments as files on anyFetch.
```
Comment (merlinND) [inline resources/hydraters.md]: AnyFetch

Comment (merlinND) [inline resources/hydraters.md]: > Each attachment is added as a new document and goes through its own hydration chain.

File: resources/hydraters.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+* Input document type: `document`
+* Output document type: `email`
+* Force-reset creation date
```
Comment (merlinND) [inline resources/hydraters.md]: > The creation date is always set to the time at which the email has been sent.

File: resources/hydraters.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+    - `to`
+    - `from`
+    - `cc`
+    - `bcc`
+    - `subject`
+    - `date`
+    - `text`
+* Output data:
+   - `html`
+* Create additional documents for each attachment
```
Comment (merlinND) [inline resources/hydraters.md]: Creates

File: resources/hydraters.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+### image.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/image.hydrater.anyfetch.com)
+
+Generate thumbnails and preview from an image.
```
Comment (merlinND) [inline resources/hydraters.md]: > Output sizes can be customized.

And image data is saved directly as base 64 in `data`?

File: resources/hydraters.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+* Input document type: `file`
+* Output document type: `image`
+* Output data:
+   - `display`
+   - `thumb`
+
+### iptc.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/iptc.hydrater.anyfetch.com)
+
+Retrieve IPTC data from an image.
```
Comment (merlinND) [inline resources/hydraters.md]: Explain in a sentence what IPTC is? I think I've heard you guys say it was not immediately easy to find info about this.

File: resources/hydraters.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+* Input document type: `file`
+* Output data:
+   - `author`
+   - `description`
+   - `keywords`
+
+### markdown.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/markdown.hydrater.anyfetch.com)
+
+Generate markdown html from a document.
```
Comment (merlinND) [inline resources/hydraters.md]: > Generate HTML from a Markdown document.

File: resources/hydraters.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+* Output data:
+    - `html` 
+
+## Document hydraters
+Hydraters working on raw JSON documents.
+
+### embedmail.hydrater.anyfetch.com
+> [Source](https://github.com/AnyFetch/embedmail.hydrater.anyfetch.com)
+
+Remove previous mail conversation from search index, to keep search accurate on new information only.
```
Comment (merlinND) [inline resources/hydraters.md:114]: > Remove the content of previous emails embedded in an email response.

It's not really removed from the search index, but rather from the metadata keys, right?

File: resources/hydraters.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+* Input document type: `email`
+* Output metadata:
+    - `text`, with stripped conversations
```
Comment (merlinND) [inline resources/hydraters.md]: > with conversations stripped

**Comment by Neamar** [comment]
> Doesn't this need to be compiled somehow? (as the endpoints' doc)
> I don't know how are the pages being served.

Jekyll handle this tedious stuff.

File: resources/hydraters.md (hunk 1 above)
Comment (Neamar) [inline resources/hydraters.md:14]: http://tika.apache.org/1.5/formats.html

Quite a lot of formats.

File: resources/hydraters.md (hunk 3 above)
Comment (Neamar) [inline resources/hydraters.md:28]: What do you mean?

Images are sent as "file" by all providers (they should act dumb), so this hydrater listen for file and transform them to image.

File: resources/hydraters.md (hunk 11 above)
Comment (Neamar) [inline resources/hydraters.md:114]: Metadata is the search index ;)

Comment (merlinND) [inline resources/hydraters.md:114]: Is it the same as the ElasticSearch search index though?
I'd rather be explicit here.

**Pull request merged**